web/src/app/licenses/licenses.component.html

```diff
@@ -0,0 +1,9 @@
+<div class="container">
+  <h1>Licenses</h1>
+
+  <ul>
+    <li>Northwoods Software: GOJS.
+      Non-comercial academic use in accordance with the terms of the license.</li>
+    <li>Material Icons. Available under Apache license version 2.0.</li>
```

Comment:
If u arę using out the app logo please add 
< a href="https://www.vecteezy.com/free-vector/logo">Logo Vectors by Vecteezy< /a>

Reply:
I am not using it yet. But I will note that for the future.